boards/stm32f4discovery/dist/flash.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+openocd -f "${RIOTBOARD}/${BOARD}/dist/openocd.cfg" \
+    -c "init" \
+    -c "targets" \
+    -c "reset" \
+    -c "halt" \
+    -c "flash protect 0 0 11 off" \
+    -c "flash erase_address 0x08000000 0x40000" \
```

Comment:
To fully erase the flash, please change to:
`-c "flash erase_address 0x08000000 0x100000" \`